libs/platform/table/table.component.ts

```diff
@@ -215,7 +215,7 @@ let tableUniqueId = 0;
         {
             directive: TableDataSourceDirective,
             inputs: ['dataSource', 'childDataSource'],
-            // eslint-disable-next-line @angular-eslint/no-outputs-metadata-property
+
```

Comment:
another empty line that doesn't really make sense (doesn't improve readability or anything)

Reply:
this was done by the tooling with lint --fix